### ISSUE TITLE
get_sweep_url() and get_project_url() are now protected (python wandb 0.10.4)

### ISSUE THREAD

**Issue by trigal** [wandb/examples#43]
with wandb 0.10.4 
https://github.com/wandb/examples/blob/7441556ef9b10340535f0a44f5d52753e77b61fe/examples/wandb-sweeps/sweeps-cross-validation/train-cross-validation.py#L62
and
https://github.com/wandb/examples/blob/7441556ef9b10340535f0a44f5d52753e77b61fe/examples/wandb-sweeps/sweeps-cross-validation/train-cross-validation.py#L63

gives this error
```AttributeError: 'Run' object has no attribute 'get_sweep_url'```

as temporary workaround one can use 
```
    sweep_url = sweep._get_sweep_url()
    project_url = sweep._get_project_url()
```

**Comment by charlesfrye** [comment]
Thanks for pointing this out and providing a workaround, @trigal! This will be resolved in the next release of `wandb`, 0.10.5, out next week.

**Comment by brando90** [comment]
https://community.wandb.ai/t/how-do-i-print-the-wandb-sweep-url-in-python/4133/2
